x-pack/docs/en/security/securing-communications/setting-up-ssl.asciidoc

```diff
@@ -9,8 +9,7 @@ WARNING: Clusters that do not have encryption enabled send all data in plain tex
 including passwords. If the {es} {security-features} are enabled, unless you have a trial license, you must configure SSL/TLS for internode-communication.
 
 The following steps describe how to enable encryption across the various
-components of the {stack}. You must perform each of the steps that are
-applicable to your cluster.
+components of the {stack}. 
```

Comment:
As the person responsible for introducing the "encrypting communications" language, I think we should instead be talking about configuring TLS. Something like:

> To configure TLS to encrypt inter-node communications on the transport layer and enable stack components to connect to Elasticsearch over HTTPS, you need to: